planex/makesrpm.py

```diff
@@ -33,16 +34,27 @@ def parse_args_or_exit(argv):
     return parser.parse_known_args(argv)
 
 
-def parse_patchseries(series_file):
+def parse_patchseries(series_file, guard = ''):
     """
     Parse series file and return the list of patches
     """
+    guard_re = re.compile(r'([\S]+)( #.*)?')
```

Comment:
Possibly, I'm struggling to find a formal definition of the patch guard format. The convention is a single space but handling any whitespace would be prudent.